packages/backend/src/db/user/userStore.service.ts

```diff
@@ -23,10 +23,23 @@ export class UserStoreService {
     return this.userModel.find().exec();
   }
 
+  async findOneByFirebaseId(firebaseId: string): Promise<UserDocument> {
+    return this.userModel.findOne({ firebaseId: firebaseId }).exec();
+  }
+
   async update(id: string, updateUserModel: Partial<UserModel>): Promise<User> {
```

Comment:
Please make sure everything returns a UserDocument
```suggestion
  async update(id: string, updateUserModel: Partial<UserModel>): Promise<UserDocument> {
```